### DISCUSSION TITLE
Is MotherDuck already Integrated?

### DISCUSSION THREAD

**Discussion by TianzeMYou** [duckdb/pg_duckdb#186] [General]
title

**Comment by wuputah** [comment]
Not yet. The necessary work is in progress!

**Reply by TianzeMYou** [reply]
Thanks @wuputah 

Also wanted to know if there are plans to backport to previous pg versions aside from 16 and 17?

**Reply by wuputah** [reply]
The project is very young and we are not yet committing to supporting older versions permanently and for all features, but there is an interested party who is adding PG15 and probably PG14 support as well, see #180 and #178.

**Comment by TianzeMYou** [comment]
When inserting into duckdb.secrets for cloud storage (e.g. S3) credentials, is this also temporary/in-memory?
https://github.com/duckdb/pg_duckdb/blob/b3d8315920b87f95dc7b17642681684eb1d758a0/README.md?plain=1#L70

**Reply by wuputah** [reply]
No, it's a standard Postgres (heap) table.

**Comment by JelteF** [comment]
It is now
